src/www/ui/css/spasht.css

```diff
@@ -8,6 +8,11 @@
 }
 
 #definitionsTable,
-#definitionsRow {
+.definitionsRow {
     border: 1px solid black;
 }
```

Comment:
This CSS looks similar to following

https://github.com/fossology/fossology/blob/8f692a66355e5c3acbd022b251ff39b7edf366a8/src/www/ui/css/fossology.css#L120-L123

Can it be reused?